jenkins/seed.groovy

```diff
@@ -0,0 +1,22 @@
+projects = [
+    [name: "hmda-platform", repo: "hmda-platform", jenkinsfilePath: "hmda/Jenkinsfile", ],
+    [name: "check-digit", repo: "hmda-platform", jenkinsfilePath: "check-digit/Jenkinsfile", ],
+    [name: "hmda-help", repo: "hmda-help", jenkinsfilePath: "Jenkinsfile", ]
```

Comment:
Can you add the `institutions-api` project as well?. It has a [Jenkinsfile](https://github.com/cfpb/hmda-platform/blob/master/institutions-api/Jenkinsfile)